components/frontend/src/subject/SubjectTitle.jsx

```diff
@@ -71,38 +74,40 @@ export function SubjectTitle({
     const subjectTitle = (atReportsOverview ? report.title + " ❯ " : "") + subjectName
     const subjectUrl = `${window.location}#${subject_uuid}`
     return (
-        <HeaderWithDetails
-            header={subjectTitle}
-            item_uuid={subject_uuid}
-            level="h2"
-            settings={settings}
-            subheader={subject.subtitle}
-        >
-            <Tabs
+        <div className="sticky" style={{ zIndex: zIndexSubjectTitle }}>
+            <HeaderWithDetails
+                header={subjectTitle}
+                item_uuid={subject_uuid}
+                level="h2"
                 settings={settings}
-                tabs={[
-                    { label: "Configuration", icon: <SettingsIcon /> },
-                    { label: "Changelog", icon: <HistoryIcon /> },
-                ]}
-                uuid={subject_uuid}
+                subheader={subject.subtitle}
             >
-                <SubjectParameters
-                    subject={subject}
+                <Tabs
+                    settings={settings}
+                    tabs={[
+                        { label: "Configuration", icon: <SettingsIcon /> },
+                        { label: "Changelog", icon: <HistoryIcon /> },
+                    ]}
+                    uuid={subject_uuid}
+                >
+                    <SubjectParameters
+                        subject={subject}
+                        subject_uuid={subject_uuid}
+                        subject_name={subjectName}
```

Comment:
Mixed snake case and camel case?

Reply:
Yeah, the frontend has a lot of that. Maybe it's best to create a separate PR for that.